src/legacy/core_plugins/kibana/public/doc_viewer/doc_viewer.tsx

```diff
@@ -44,6 +44,10 @@ export function DocViewer(renderProps: DocViewRenderProps) {
     };
   });
 
+  if (!tabs.length) {
+    return <div>No Tabs registered</div>;
```

Comment:
Should be i18n-ified.

Reply:
In this case, it's an internal comment, because the state of no tabs registered, is only relevant for testing, can't happen in 'real' life, it think I'll replace with returning `null` and adding a comment to be more precise, thx!

Reply:
changed thx!